nucypher/blockchain/eth/actors.py

```diff
@@ -210,8 +209,8 @@ def __init__(
         )
 
         # TODO: registry usage (and subsequently "network") is inconsistent here
-        coordinator_network_registry = InMemoryContractRegistry.from_latest_publication(
-            network=coordinator_network
+        coordinator_network_registry = ContractRegistry.from_latest_publication(
+            domain=coordinator_network
```

Comment:
Should this be `self.network` instead of `coordinator_network`? We no longer obtain registries via the l2 network.

Reply:
Good eye.  I'll have a closer look. 👀